java/vector/src/main/java/org/apache/arrow/vector/FixedSizeBinaryVector.java

```diff
@@ -116,6 +117,18 @@ public byte[] get(int index) {
     return dst;
   }
 
+  /**
+   * Read the value at the given position to the given output buffer.
+   * The caller is responsible for checking for nullity first.
+   *
+   * @param index position of element.
+   * @param outputBuffer the buffer to write into.
+   */
+  public void readBytes(int index, ReusableBuffer<?> outputBuffer) {
```

Comment:
Why readBytes vs just overloading getObject? Or just "read"?

Reply:
I'll go with read(). Avoiding using getObject() since it sounds too much like it should return a value vs. populate a buffer.